schemas/2018-12-01/Microsoft.Network.json

```diff
@@ -89,6 +89,67 @@
       ],
       "description": "Microsoft.Network/applicationGateways"
     },
+    "ApplicationGatewayWebApplicationFirewallPolicies": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "Microsoft.Network/ApplicationGatewayWebApplicationFirewallPolicies"
```

Comment:
>ApplicationGatewayWebApplicationFirewallPolicies [](start = 31, length = 48)

do we need to add it to the root schemas?

Reply:
updated